### PR TITLE
[New Actions] Ghost Admin

### LINKED ISSUE #3286: Ghost.org (Admin API)
### App
Ghost.org (Admin API)
https://ghost.org/docs/admin-api/

### Triggers
- Member created
- Member deleted

### Actions
- Create member
- Update member

### PR DESCRIPTION
**Pull request by vellames-turing**
Closes #3286 

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/3408"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/ghost_org_admin_api/actions/create-member/create-member.mjs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import app from "../../ghost_org_admin_api.app.mjs";
+
+export default {
+  key: "ghost_org_admin_api-create-member",
+  name: "Create Member",
+  description: "Create a new member in Ghost. [See the docs here](https://ghost.org/docs/admin-api/#members)",
```
Comment (feyzullah) [inline components/ghost_org_admin_api/actions/create-member/create-member.mjs:6]:
```suggestion
  description: "Create a new member in Ghost. [See the docs here](https://ghost.org/docs/admin-api/#creating-a-member)",
```
Can we update anchor links?

File: components/ghost_org_admin_api/actions/update-member/update-member.mjs [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import app from "../../ghost_org_admin_api.app.mjs";
+
+export default {
+  key: "ghost_org_admin_api-update-member",
+  name: "Update Member",
+  description: "Update a member in Ghost. [See the docs here](https://ghost.org/docs/admin-api/#members)",
```
Comment (feyzullah) [inline components/ghost_org_admin_api/actions/update-member/update-member.mjs:6]:
```suggestion
  description: "Update a member in Ghost. [See the docs here](https://ghost.org/docs/admin-api/#updating-a-member)",
```

**Review by feyzullah — APPROVED** [review]
Hi @vellames-turing, I just added anchor links, LGTM!

**Pull request merged**